pyproject.toml

```diff
@@ -37,6 +37,7 @@ mail-parser = "^3.15.0"
 requests = "^2"
 lxml = "^4.9.3" # TODO: remove when https://github.com/adbar/trafilatura/issues/449 is fully resolved
 pillow = "^10.1.0"
+google = "^3.0.0"
```

Comment:
Is this change intentional?

Reply:
As stated above, without this dependency the python 3.11 pipeline failed for me - but I will remove the dependency and see if the pipeline runs successfully

Reply:
Ah sorry missed the previous review comment! Yeah let's try again without it, I can help troubleshoot if it still gives issues.